core/src/main/scala/org/apache/spark/sql/delta/Checkpoints.scala

```diff
@@ -335,6 +350,15 @@ object Checkpoints extends DeltaLogging {
     CheckpointMetaData(snapshot.version, checkpointSize.value, None)
   }
 
+  private[delta] def isGCSPath(hadoopConf: Configuration, path: Path): Boolean = {
```

Comment:
This is a hack to detect GCS path because we cannot access GCS libraries in `core`.